model/network.yaml

```diff
@@ -25,6 +25,10 @@ groups:
           [OSI Transport Layer](https://osi-model.com/transport-layer/) or
           [Inter-process Communication method](https://en.wikipedia.org/wiki/Inter-process_communication).
           The value SHOULD be normalized to lowercase.
+
+          Consider always setting the transport when setting a port number, since
```

Comment:
I wonder if such clarification makes also sense on the `port` attributes, such as [server.port](https://github.com/open-telemetry/semantic-conventions/blob/4bbb8c907402caa90bc077214e8a2c78807c1ab9/model/server.yaml#L20C13-L20C13).

Just thinking, if such notice is only present here, when I'm looking at the port attribute I might miss that I should also add the protocol 🤔

Reply:
> I wonder if such clarification makes also sense on the port attributes, such as [server.port](https://github.com/open-telemetry/semantic-conventions/blob/4bbb8c907402caa90bc077214e8a2c78807c1ab9/model/server.yaml#L20C13-L20C13).

To be honest, I initially wanted to put it there but that would be so much repetition ((server|client|source|destination).port, (server|client).socket.port.

Reply:
(Keeping it unresolved for now, it's a good thing to note; should not block merging from my perspective and unless others feel strongly about it I will keep it as-is)